config/services.yml

```diff
@@ -258,6 +258,90 @@ https://cmr.earthdata.nasa.gov:
       - image: !Env ${CMR_GRANULE_LOCATOR_IMAGE}
       - image: !Env ${PODAAC_L2_SUBSETTER_IMAGE}
 
+  - name: podaac/concise
+    data_operation_version: '0.12.0'
+    type:
+      <<: *default-argo-config
+      params:
+        <<: *default-argo-params
+        image: !Env ${PODAAC_CONCISE_IMAGE}
+        image_pull_policy: !Env ${PODAAC_CONCISE_IMAGE_PULL_POLICY}
+        env:
+          <<: *default-argo-env
+          STAGING_PATH: public/podaac/concise
+    umm_s:
+      - S2153799015-POCLOUD
+    collections: []
+    capabilities:
+      concatenation: true
+      subsetting:
+        bbox: false
+        variable: false
+      output_formats:
+        - application/netcdf # Incorrect mime-type, remove when no longer needed
+        - application/x-netcdf4
+    steps:
+      - image: !Env ${CMR_GRANULE_LOCATOR_IMAGE}
+      - image: !Env ${PODAAC_CONCISE_IMAGE}
+        operations: ['concatenate']
+
+  - name: podaac/l2-subsetter-concise
+    data_operation_version: '0.12.0'
+    type:
+      <<: *default-argo-config
+      params:
+        <<: *default-argo-params
+        image: !Env ${PODAAC_CONCISE_IMAGE}
+        image_pull_policy: !Env ${PODAAC_CONCISE_IMAGE_PULL_POLICY}
+        template_type: chaining
+        env:
+          <<: *default-argo-env
+          STAGING_PATH: public/podaac/l2-subsetter-concise
+    collections:
```

Comment:
Seems like moving this to use `umm_s` as well is a good idea in the future.

Reply:
I wasn't sure if that was possible with a chain?

Reply:
I thought there was an idea that each chain would be its own UMM-S record, but not sure where that ended up in practice. @bilts or @asteiker would know best.